Resources/Prototypes/silicon-laws.yml

```diff
@@ -499,7 +499,7 @@
   - Nutimov5
   obeysTo: laws-owner-crew
 
-# ion storm random lawsets
+ #  ion storm random lawsets
```

Comment:
Unresolved.
An unnecessary change to upstream file is still here, but now with another extra space.
```suggestion
# ion storm random lawsets
```

should fix this